subprojects/docs/src/docs/userguide/gradle_wrapper.adoc

```diff
@@ -238,3 +238,41 @@ distributionSha256Sum=371cb9fbebbe9880d147f59bab36d61eee122854ef8c9ee1ecf12b8236
 ----
 
 Gradle will report a build failure in case the configured checksum does not match the checksum found on the server for hosting the distribution. Checksum Verification is only performed if the configured Wrapper distribution hasn't been downloaded yet.
+
+[[wrapper_checksum_verification]]
+== Verifying the checksum of the Gradle Wrapper JAR
+
+Since the Wrapper JAR file is usually checked in to a project's version control system, verifying its checksum provides a simple way to check its integrity.
```

Comment:
I don't understand the connection between these two clauses. How is the fact the wrapper JAR is checked into VCS relevant to verifying its checksum for integrity? I would drop the first clause unless there's a good reason to keep it.